server/src/main/java/com/android/example/KeyAttestationExample.java

```diff
@@ -249,13 +250,28 @@ private static void verifyCertificateChain(X509Certificate[] certs)
               + " should check the certificate revocation lists using the distribution points that"
               + " are listed in the intermediate and root certificates.");
     } else {
-      System.out.println(
-          "The root certificate is NOT correct. The attestation was probably generated by"
-              + " software, not in secure hardware. This means that, although the attestation"
-              + " contents are probably valid and correct, there is no proof that they are in fact"
-              + " correct. If you're using a production-level system, you should now treat the"
-              + " properties of this attestation certificate as advisory only, and you shouldn't"
-              + " rely on this attestation certificate to provide security guarantees.");
+      X509Certificate secureRoot =
+          (X509Certificate)
+              CertificateFactory.getInstance("X.509")
+                  .generateCertificate(
+                      new ByteArrayInputStream(GOOGLE_ROOT_CERTIFICATE.getBytes(UTF_8)));
```

Comment:
That's supposed to reference the newly added CA, isn't it?

```suggestion
                      new ByteArrayInputStream(GOOGLE_ROOT_CERTIFICATE_2.getBytes(UTF_8)));
```

Would be great to also have this PR include a test attestation certificate chain which roots in `GOOGLE_ROOT_CERTIFICATE_2`.

Reply:
@veehaitch yes GOOGLE_ROOT_CERTIFICATE_2 is second one . I will also add the test cert chain for the 2nd certificate.